frontend/pages/dashboard/new-post.tsx

```diff
@@ -87,6 +139,35 @@ const NewPostPage: NextPage = () => {
             onChange={setLangId}
           />
 
+          <div className="header-preview-container">
+            <PostHeader
+              postTitle={title}
+              postStatus={PostStatusType.Published}
+              publishDate={new Date().toISOString()}
+              authorName={currentUser?.name || 'anonymous'}
+              postImage={image.largeSize || '/images/samples/sample-post-img.jpg'}
+            >
+              <div className="header-preview-options">
+                <Button
```

Comment:
I saw in the PR description GIF that we could potentially benefit from adding a value for the `loading` prop here on `<Button />`. We could add something like `const [uploadingImage, setUploadingImage] = useState(false)` and set it to true when uploading the file and then use

Reply:
Yep yep!! I was actually wanting to add that in right after work! Wasn't 100% sure how best to implement it yet so I'll try your suggestion!

Reply:
Done!!! ✅